src/plugins/ops.ts

```diff
@@ -236,6 +243,19 @@ export function addMutatingInfix(
   };
 }
 
+export const addPostfixIncAndDec: Plugin = {
+  name: "addPostfixIncAndDec",
+  visit(node) {
+    if (
+      node.kind === "MutatingInfix" &&
+      ["+", "-"].includes(node.name) &&
```

Comment:
Note the return value of `(++x)` is the same as the return value of `(x+=1)` and `(x++)+1`. So prefix might be more appropriate -- I'm not sure where this gets used.

Reply:
I was thinking we will need dedicated node for inc/dec to handle more golfing anyway. Currently the return is always discarded.